### PR TITLE
Styled logo for mobile

### PR DESCRIPTION
**Pull request by williammadisondavis**
I adjusted the .logo to work for the mobile format.
@clinturbin @running-on-sunshine

### CODE REVIEW

File: stylesheets/styles.css [hunk 1]
```diff
@@ -19,8 +19,8 @@ body, html {
    
 .logo {
     display: flex;
-    height: 180%;
-    width: 82rem;
+    height: 120%;
+    width: 35rem;
 }
```
Comment (running-on-sunshine) [inline stylesheets/styles.css]: Thoughts: 

- The logo seems to fit most devices, which is great! 🙂 However, it doesn't seem to fit iPhone 5/SE or smaller-screened devices. Suggestion: 
```
.logo {
    display: flex;
    height: 100%;
    width: 30rem;
}
```
- Could we possibly change the height to 100%? As it currently stands at 120%, the logo is overflowing out of the 'header' container.

Comment (clinturbin) [reply stylesheets/styles.css]: I agree. Changing the height to 100% fixes the display issues on smaller devices.

**Comment by running-on-sunshine** [comment]
Thoughts: 

- Looks good! 🙂 Thanks for updating the logo's size to accommodate for smaller-screened devices!
- We might want to take a look at stying the background color for the header in the future, unless we are cool with beige! 

You may proceed to merge! 👍

**Comment by clinturbin** [comment]
I agree, it looks good to merge!👍

**Pull request merged**